[19.09] add caret to toolsheds dropdown in admin menu

martenson opened the pull request:
so it is clear this is a menu. cc @natefoo 

before
![Galaxy](https://user-images.githubusercontent.com/1814954/67320799-b3522b80-f4dc-11e9-97c1-c48d089100b7.png)

after
![Galaxy](https://user-images.githubusercontent.com/1814954/67320828-bd742a00-f4dc-11e9-99a1-bb329e6942ab.png)

martenson committed:
add caret to toolsheds dropdown in admin menu

so it is clear this is a menu

mvdbeek commented:
Could we make this more button-ish maybe ? I feel like that's still easy to miss (but definitely better!)

martenson committed:
improve UX of clearing search terms

by providing tooltip,
and removing the disabled trigger, so it gets a natural tabindex

martenson commented:
@mvdbeek Button feels too dominant/heavyhanded for such a niche function. I think it is enough to have the caret.


![Galaxy](https://user-images.githubusercontent.com/1814954/67323486-830c8c00-f4e0-11e9-9916-f47d3956d41f.png)

mvdbeek commented:
yeah, 👍

dannon commented:
Thanks for this.  I had initially missed it was a dropdown for selecting the toolshed as well.

guerler commented:
Thank you for fixing this. We could also re-use what we have for the installation monitor in #8810. Its a `fa fa-angle-double-down` in front of the link. Otherwise we can revisit this another time. Thank you.

martenson commented:
@guerler Oh, I did not notice that. I thought of carets as our 'standard' given we use them in masthead and all select2 dropdowns including the toolform. 

This feels more like a menu than the collapsible status you implemented for the monitor. So I am happy with the caret here and the double angle there I think.

guerler commented:
Cool, that works. Thank you.

natefoo reviewed (approved):
Thanks @martenson!
